geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/TiledRasterRDD.scala

```diff
@@ -429,6 +436,25 @@ class SpatialTiledRasterRDD(
     SpatialTiledRasterRDD(None, multibandRDD)
   }
 
+  def hillshade(
+    sc: SparkContext, 
```

Comment:
Just noticing this now but we don't need `sc` here, we can and should just pull it from `rdd.sparkContext`